samples/EntityFramework/HttpTrigger.cs

```diff
@@ -0,0 +1,75 @@
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Hollan.Function
+{
+    public class HttpTrigger
+    {
+        private readonly BloggingContext _context;
+        public HttpTrigger(BloggingContext context)
+        {
+            _context = context;
+        }
+
+        [Function("GetPosts")]
+        public HttpResponseData GetPosts(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "posts")] HttpRequestData req,
+            FunctionContext context)
+        {
+           var logger = context.GetLogger("GetPosts");
+           logger.LogInformation("C# HTTP GET/posts trigger function processed a request.");
+
+           var postsArray = _context.Posts.OrderBy(p => p.Title).ToArray();
+           var response = req.CreateResponse(HttpStatusCode.OK);
+           response.WriteString(JsonConvert.SerializeObject(postsArray));
```

Comment:
Use the new JSON extension APIs (`WriteAsJsonAsync`)

Reply:
@fabiocav has that been published yet? It's not showing for me in the `preview5`

Reply:
Ah, yeah, I think the samples were updated to use a package reference here.

Will look at the updates, but I see you had a follow up. Did you update the reference to get the new APIs?

Reply:
OH can do - do we prefer we use a package reference to the source or keep it referencing NuGet?  I'm ok either way, only benefit of the latter is you can "copy paste" out of the repo and it all still works without updating references, but I can reference to base.  LMK what you prefer